src/sphinx_book_theme/theme/sphinx_book_theme/components/sidebar-logo.html

```diff
@@ -5,7 +5,7 @@
         {% set logo_url=logo %}
       {% endif %}
       {% if logo_url %}
-      <img src="{{ pathto('_static/' + logo_url, 1) }}" class="logo" alt="logo">
+      <img src="{{ logo_url|e }}" class="logo" alt="logo">
```

Comment:
Why is this being escaped? Isn't it an already-encoded URL?

Reply:
@pradyunsg I don't think that's guaranteed since the `logo_url` value comes from the user in conf.py and the code I linked to above doesn't seem to be escaping anything.

Reply:
You're correct and I have to go fix something in Furo. :)

x-ref: https://github.com/sphinx-doc/sphinx/blob/6e6b9ad45194177811551c77f5f4040213d1c661/sphinx/themes/basic/layout.html#L56